### PR TITLE
Add compat data for Slotable api

### PR DESCRIPTION
**Pull request by lucian95**

### CODE REVIEW

File: api/Slotable.json [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+              "version_added": "53"
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": true
```
Comment (ExE-Boss) [inline api/Slotable.json]: This should be:
```json
{
  "version_added": "58",
  "flags": [
    {
      "type": "preference",
      "name": "dom.webcomponents.enabled",
      "value_to_set": "true"
    }
  ]
}
```

File: api/Slotable.json [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+            },
+            "firefox_android": {
+              "version_added": true
```
Comment (ExE-Boss) [inline api/Slotable.json]: ditto

File: api/Slotable.json [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "dom.webcomponents.enabled",
+                  "value_to_set": "true"
+                }
+              ]
+            },
+            "firefox_android": {
+              "version_added": true
```
Comment (ExE-Boss) [inline api/Slotable.json]: You missed the information for Firefox for Android, this should be the same as in Firefox for Desktop.

Comment (lucian95) [reply api/Slotable.json]: Right, my bad

**Comment by lucian95** [comment]
Sorry for the messiness, I forgot to pull before pushing. And thanks for the explanation; I'm still learning and it's a huge help.

**Comment by chrisdavidmills** [comment]
No worries! It is a complex process; it took me a while to get used to as well. And the Web components support data is not the simplest. Thanks for the contribution ;-)

**Pull request merged**